fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/api/content/registry/v1/TillableBlockRegistry.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.content.registry.v1;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+
+import net.fabricmc.fabric.api.content.registry.v1.util.Taggable2ObjectMap;
+import net.fabricmc.fabric.impl.content.registry.ToolBlockTransformationRegistries;
+
+/**
+ * Registry of Blocks that when tilled turn into a specific BlockState.
+ */
+public interface TillableBlockRegistry extends Taggable2ObjectMap<Block, BlockState> {
+	Taggable2ObjectMap<Block, BlockState> INSTANCE = ToolBlockTransformationRegistries.TILLABLE_BLOCKS;
```

Comment:
If you have many classes holding one map, I'd suggest you move them into one class. That is way better than your "hey I reduced mixin classes" for making api more efficient.

Reply:
That would lead to some awful and unintuitive names. I brought it up in the #api channel when working on this and we decided individual classes was better than coming up with a crap name that makes sense for all of them. API class count is not an issue.

Reply:
so you put them in a `ContentRegistry` class and have fields named like `TILLABLE_BLOCKS` `SHOVELABLE_BLOCKS` etc. How is that unintuitive or awful at all?

Reply:
Hmm that could work. It doesn't work quite so well for things like the flammable block registry which uses its own interface, though. Worth considering, though.

Reply:
Like you can consider grouping block -> block transformations (hoe, axe, shovel) in a class, block properties (fire) in a class, item properties (fuel, brewing stand fuel etc) in another class, etc. etc.